addons/website/tests/test_qweb.py

```diff
@@ -107,3 +107,86 @@ def test_process_att_url_crap(self):
             match.reset_calls()
             self._test_att('/x?y#z', {'href': '/x?y#z'})
             match.assert_called_with('/x', method='POST', query_args='y')
+
+class TestQwebDataSnippet(TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.env['ir.ui.view'].create({
+            'name': 'some_html',
+            'type': 'qweb',
+            'key': 'website.some_html',
+            'arch': '''
+                <t t-name="some_html"><article><span>Hello</span><t t-out="0"/></article></t>
```

Comment:
Is it a problem to make those on multiple lines with proper indentation? Note that you can remove the indentation at the start of the line if you did that for printing purposes.
```python
        self.env['ir.ui.view'].create({
            'name': 'some_html',
            'type': 'qweb',
            'key': 'website.some_html',
            'arch': '''
<t t-name="some_html">
    <article>
</t>
            '''
```

Reply:
EDIT: in fact I think you don't even need to indent weirdly like I showed but then use `_assertXmlEqual` to compare the string :+1: